modules/vaos/spec/request/appointment_requests_request_spec.rb

```diff
@@ -10,80 +10,127 @@
     allow_any_instance_of(VAOS::UserService).to receive(:session).and_return('stubbed_token')
   end
 
-  context 'loa1 user with flipper enabled' do
-    let(:current_user) { build(:user, :loa1) }
-
-    it 'does not have access' do
-      get '/v0/vaos/appointment_requests'
-      expect(response).to have_http_status(:forbidden)
-      expect(JSON.parse(response.body)['errors'].first['detail'])
-        .to eq('You do not have access to online scheduling')
-    end
-  end
-
-  context 'loa3 user' do
-    let(:current_user) { build(:user, :mhv) }
-    let(:start_date) { Date.parse('2019-08-20') }
-    let(:end_date) { Date.parse('2020-08-22') }
-    let(:params) { { start_date: start_date, end_date: end_date } }
+  describe 'GET /v0/vaos/appointment_requests' do
```

Comment:
NOTE: most of these specs were just tabbed over, not actually modified. Wrapped them in a new describe block. GET requests are unchanged as part of this PR.